app/views/members/_search.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_tag(members_path, method: :get) do %>
+  <%= text_field_tag :query, params[:query] %>
```

Comment:
>  I couldn't figure out how to sanitize the query for display in the search box.

Another possible approach might be, rather than have to try to remember/possibly forget to use the `sanitize` method in multiple places, create a helper somewhere which handles the strong parameters support, input sanitization, etc, and then reuse that method in various locations. 

Also, for clarification _(because I did poorly the first time)_:

> _don't forget to ensure the proper sanitization scope_

This refers to the context of where that the data/content is being used _(e.g., HTML vs. CSS vs. JS vs. ...)_. For example, simply using an HTML sanitizer and then dropping content into a JS context is not sufficient.